pyfstat/mcmc_based_searches.py

```diff
@@ -1,5 +1,4 @@
-""" Searches using MCMC-based methods """
-
+""" PyFstat search & follow-up classes using MCMC-based methods """
```

Comment:
Let's already but it here centrally:
```suggestion
"""PyFstat search & follow-up classes using MCMC-based methods

The general approach is described in
Ashton & Prix (PRD 97, 103020, 2018):
https://arxiv.org/abs/1802.05450
and we use the `ptemcee` sampler
described in Vousden et al. (MNRAS 455, 1919-1937, 2016):
https://arxiv.org/abs/1501.05823
and based on Foreman-Mackey et al. (PASP 125, 306, 2013):
https://arxiv.org/abs/1202.3665
"""
```